src/Components/Common/PageTitle.tsx

```diff
@@ -70,15 +70,15 @@ export default function PageTitle({
             >
               <CareIcon
                 icon="l-angle-left"
-                className="border-box mr-1 rounded-md text-5xl hover:bg-secondary-200"
```

Comment:
Can we wrap only the necessary components under `Page`/`PageTitle` component so that this change wouldn't be necessary?

Ideally Page/PageTitle component shouldn't be present as a child of `PrintPreview` component right.

Reply:
Got it.

Reply:
If I don't add print-hidden on line 73, the button will appear on the printed document.

![image](https://github.com/user-attachments/assets/eec55720-9c85-4572-8ffd-14f6a4daeebc)